### ISSUE TITLE
Netflix Geoblocking bypass . DNS LEAK Problem..

### ISSUE THREAD

**Issue by plbartek89** [trailofbits/algo#1339]
Hello , 
I Try to bypass Netflix Geoblocking and  installed my algo vpn server on my Polish VPS server. (Want to use Polish Netflix Content outside of Poland).
In config.cfg nothing changed except username , when i go to whoer.ner it shows only : Your anonymity: 55% DNS different. 
Dnsleaktest : [Click](https://gyazo.com/b4eaae4d90a5b7a9c128642d007be616)

How to fix that ??
Thanks

**Comment by TC1977** [comment]
Some specifics needed here. What client are you trying to connect with, are you using IPsec (strongswan) or Wireguard, and what is Deutsche Telekom AG?

I've had my home Wi-Fi router set to use my Algo server as its DNS, so everyone on my home LAN can enjoy adblocking. Sometimes I would get similar DNS leaks, which were caused by the upstream home router (supplied by the ISP) using the ISP's DNS servers. Since I couldn't change those defaults, I changed the DNS settings on my home Wi-Fi router to point to the Algo server and itself (at "192.168.x.y") as a default. I wonder if your home ISP-supplied router is doing something similar.
